src/osuTK.iOS/IOSAppDelegate.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using UIKit;
+using Foundation;
+using osuTK.Input;
+using osuTK.Platform;
+
+namespace osuTK.iOS
+{
+    public abstract class IOSAppDelegate : UIApplicationDelegate
+    {
+       
+        public abstract override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions); // Exposing FinishedLaunching for osu-framework to handle
```

Comment:
this is never being called?

Reply:
It is in osu-framework, that’s a change that is needed for the game to boot at all [osu-framework#2048](
https://github.com/ppy/osu-framework/pull/2048/commits/7e333c759156a47cedb7a2b4460e321fe7f1db88)

Reply:
Then include it in osu-framework, not here...